### ISSUE TITLE
new function dxCreateCustomTexture(...)

### ISSUE THREAD

**Issue by Petya0x44** [multitheftauto/mtasa-blue#1230]
I suggest creating a function for create custom texture (from gta sa). This function can be used, for example, to design one object with the texture of another object. The advantage of this function is that there is no need to download the txd file, since these textures are already present in GTA SA.

**Syntax**
`dxCreateCustomTexture(string txd_name, string texture_name)`

**Example**
```
texture = dxCreateCustomTexture("des_se2", "des_dirt1")
dxSetShaderValue(shader, "tex", texture)
engineApplyShaderToWorldTexture(shader, "vehiclegeneric256")
```

**Comment by AfuSensi** [comment]
I think this functionality is already implemented in https://github.com/multitheftauto/mtasa-blue/pull/1058

**Comment by Petya0x44** [comment]
> I think this functionality is already implemented in #1058

Yes, this is analogous to what i suggest. Sorry i didn't see this issue

**Comment by qaisjp** [comment]
Duplicate of #1040
